powerlevel9k.zsh-theme

```diff
@@ -1023,9 +1023,9 @@ prompt_load() {
   # Replace comma
   load_avg=${load_avg//,/.}
 
-  if [[ "$load_avg" -gt $(bc -l <<< "${cores} * 0.7") ]]; then
```

Comment:
These are changes to `prompt_load` so muddies your PR. Would it be possible to do this in a separate PR if you want to suggest these changes, and also document what the changes are and why?

Reply:
That have to be a slip. Wasn't my intention to touch this. I will revert this immediately.